core/services/ardupilot_manager/main.py

```diff
@@ -68,4 +104,4 @@ def remove_endpoints(response: Response, endpoints: Set[Endpoint] = Body(...)) -
 
 if __name__ == "__main__":
     autopilot.run()
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    uvicorn.run(app, host="0.0.0.0", port=8000, log_config=None)
```

Comment:
Why log_config is None ? Can you add a comment explaining ?

Reply:
Because we are capturing the uvicorn logging output to loguru, so we disable output by uvicorn itself.
Do you mean comment on the code or the commit message?

Reply:
Added a comment on the line before.